app/views/bgeigie_imports/_head.html.erb

```diff
@@ -13,6 +13,14 @@
           <%- end -%>
         </li>
       <%- end -%>
+      <%- if !@bgeigie_import.awaiting_approval? && user_signed_in? && current_user.moderator? && !@bgeigie_import.done? -%>
```

Comment:
@seanbonner definitely set to show only if drive is not awaiting approval (presumably approved). Should the condition be just visible to any moderator?